src/main/java/com/prgrms/shorturl/url/controller/validation/HttpsUrlValidator.java

```diff
@@ -0,0 +1,15 @@
+package com.prgrms.shorturl.url.controller.validation;
+
+import jakarta.validation.ConstraintValidator;
+import jakarta.validation.ConstraintValidatorContext;
+
+public class HttpsUrlValidator implements ConstraintValidator<HttpUrl, String> {
+    @Override
+    public boolean isValid(String value, ConstraintValidatorContext context) {
+        if (value == null || value.isBlank()) {
+            return true;
+        }
+        return value.startsWith("https://") || value.startsWith("http://");
+    }
+}
+
```

Comment:
```suggestion
public class HttpsUrlValidator implements ConstraintValidator<HttpUrl, String> {
    
    @Override
    public boolean isValid(String value, ConstraintValidatorContext context) {
        if (value == null || value.isBlank()) {
            return true;
        }
    
        return value.startsWith("https://") || value.startsWith("http://");
    }
}

```
적절한 개행은 가독성을 향상시킬수있어요